Persistence/scheduled task creation

```diff
@@ -0,0 +1,5 @@
+//Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_susp_schtask_creation.yml
+//Questions via Twitter: @janvonkirchheim 
+
+ProcessCreationEvents 
+| where ((FolderPath endswith "\\schtasks.exe" and ProcessCommandLine contains " /create ") and (AccountName != "system"))
```

Comment:
I think all these parenthesis are redundant, since all conditions are combined with an `and`.